rules_go_gapic/go_gapic.bzl

```diff
@@ -15,14 +15,24 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
 load("@com_google_api_codegen//rules_gapic:gapic.bzl", "proto_custom_library", "unzipped_srcjar")
 
-def go_gapic_library(name, srcs, importpath, deps, release_level="", grpc_service_config = None, **kwargs):
+def go_gapic_library(
+  name,
+  srcs,
+  importpath,
+  deps,
+  release_level = "",
+  grpc_service_config = None,
+  service_yaml = None,
+  **kwargs):
+
   output_suffix = ".srcjar"
   file_args = {}
 
   if grpc_service_config:
-    file_args = {
-      grpc_service_config: "grpc-service-config",
-    }
+    file_args[grpc_service_config] =  "grpc-service-config"
+
+  if service_yaml:
+    file_args[service_yaml] = "gapic-service-config"
```

Comment:
Service config yaml is not gapic-specific (i.e. makes sense and existed before gapic became a thing). Is there a reason why "gapic-" prefix is used here?

Reply:
Yeah, the thought was that if other microgenerators needed the service yaml, they'd all key in on the name `gapic-service-config`, the "gapic-" prefix making it language agnostic (as opposed to `go-gapic-package` for example which should not be used for a python package). Luke and I settled on it, but this option is going to be removed once I figure out how to get what I need in it (some doc strings) from another source, so I'm not too worried about the name. It is also already being used though, so if possible I'd like not to change it.